backends/hip-ref/ceed-hip-ref-basis.c

```diff
@@ -66,13 +66,9 @@ int CeedBasisApply_Hip(CeedBasis basis, const CeedInt num_elem, CeedTransposeMod
       CeedCallBackend(CeedRunKernelDim_Hip(ceed, data->Weight, num_elem, block_size_x, block_size_y, 1, weight_args));
     } break;
     // LCOV_EXCL_START
-    // Evaluate the divergence to/from the quadrature points
     case CEED_EVAL_DIV:
-      return CeedError(ceed, CEED_ERROR_BACKEND, "CEED_EVAL_DIV not supported");
-    // Evaluate the curl to/from the quadrature points
     case CEED_EVAL_CURL:
-      return CeedError(ceed, CEED_ERROR_BACKEND, "CEED_EVAL_CURL not supported");
-    // Take no action, BasisApply should not have been called
+      return CeedError(ceed, CEED_ERROR_BACKEND, "%s not supported", CeedEvalModes[eval_mode]);
     case CEED_EVAL_NONE:
       return CeedError(ceed, CEED_ERROR_BACKEND, "CEED_EVAL_NONE does not make sense in this context");
```

Comment:
TBH, I don't know what the context is here, but it might be helpful to expand the error message here to be more direct. Looks like other functions, including CUDA, have a similar error message.

Reply:
I think the main thought in my head is that the message is vague enough to have the immediate follow up question of "why does it not make sense". I think I understand the context now. Given that the error is probably seen by backend developers and not end users, maybe

```
CEED_EVAL_NONE should be handled outside this routine.
```

Reply:
Honestly, its trivial to add this functionality so we don't need this error message

Reply:
How about 9ea2cfd987bb00fe9579c83295ba7c97d225c170 for bypassing the issue?